### ISSUE TITLE
Add support for TB

### ISSUE THREAD

**Issue by erinyoung** [UPHL-BioNGS/Grandeur#135]
We don't sequence TB at UPHL, but there seems to be increasing interest in using Grandeur for TB.

Jumping point : https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001081

- Some accessions : https://raw.githubusercontent.com/mbhall88/drprg-paper/main/config/illumina.samplesheet.pass.csv
- https://github.com/mbhall88/drprg
- https://github.com/mbhall88/drprg-index/tree/main

**Comment by DrB-S** [comment]
Adding drprg would be helpful to show specific drug susceptibility, and would hopefully mimic what varpipe-wgs provides.  Here is some example varpipe-wgs drug susceptibility output:


Sample ID | Drug | Variant | Interpretation
-- | -- | -- | --
TB-0088 | INH | No reportable variant detected | INH-S
TB-0088 | RIF | No reportable variant detected | RIF-S
TB-0088 | PZA | No reportable variant detected | PZA-S
TB-0088 | FQ | No reportable variant detected | FQ-S
TB-0088 | EMB | No reportable variant detected | EMB-S
TB-0089 | INH | No reportable variant detected | INH-S
TB-0089 | RIF | No reportable variant detected | RIF-S
TB-0089 | PZA | No reportable variant detected | PZA-S
TB-0089- | FQ | No reportable variant detected | FQ-S
TB-0089 | EMB | No reportable variant detected | EMB-S
TB-0090 | INH | katG_p.Arg463Leu | INH-S
TB-0090 | RIF | No reportable variant detected | RIF-S
TB-0090 | PZA | No reportable variant detected | PZA-S
TB-0090 | FQ | No reportable variant detected | FQ-S
TB-0090 | EMB | embB_p.Glu378Ala | EMB-S
TB-0092 | INH | katG_p.Arg463Leu,katG_p.Asp94Asn | INH-U
TB-0092 | RIF | No reportable variant detected | RIF-S
TB-0092 | PZA | No reportable variant detected | PZA-S
TB-0092 | FQ | gyrA_p.Asp94Gly | FQ-R
TB-0092 | EMB | No reportable variant detected | EMB-S

**Comment by erinyoung** [comment]
Added a few reference genomes in https://github.com/UPHL-BioNGS/Grandeur/pull/138, but Grandeur doesn't check for any matches and no TB or NTM tools were added.

Tools that I'm looking into for a future update:
- [tb-profiler](https://github.com/jodyphelan/TBProfiler) - starts from reads
- [varpipe](https://github.com/CDCgov/NCHHSTP-DTBE-Varpipe-WGS) - starts from reads
- [mykrobe](https://github.com/Mykrobe-tools/mykrobe)
- [Dr. PRG](https://github.com/mbhall88/drprg) - allows contigs, but most of the documentation is for reads

**Comment by erinyoung** [comment]
WHO publication : https://www.who.int/publications/i/item/9789240082410

**Comment by erinyoung** [comment]
https://www.ijidonline.com/article/S1201-9712(23)00771-3/fulltext#seccesectitle0009 used TB Profiler

**Comment by erinyoung** [comment]
https://github.com/UPHL-BioNGS/Grandeur/pull/153 has mykrobe and drprg modules incorporated into the workflow. The results of mykrobe are also in the final summary file - mostly because there are already results in csv format.

In practice, we've found mykrobe to be more useful in Nontuberculous mycobacteria (NTM) than other tools.

We've found Dr. PRG to be more useful than mykrobe with TB. The drug resistance predictions from Dr. PRG are in a json file (mykrobe also has a json file with predictions) and those files are available in the tool's subdirectory.

This all needs to be added to the documentation.
